manifests/init.pp

```diff
@@ -26,15 +26,24 @@
 # @param in_ssh
 #   Allow inbound to ssh servers.
 #
+# @param reject_with
+#   How to discard packets not matching any rule. If `false`, the
+#   fate of the packet will be defined by the chain policy (normally
+#   drop), otherwise the packet will be rejected with the REJECT_WITH
+#   policy indicated by the value of this parameter.
+#
 class nftables (
-  Boolean $in_ssh    = true,
-  Boolean $out_ntp   = true,
-  Boolean $out_dns   = true,
-  Boolean $out_http  = true,
-  Boolean $out_https = true,
-  Boolean $out_all   = false,
-  Hash $rules        = {},
-  String $log_prefix = '[nftables] %<chain>s Rejected: ',
+  Boolean $in_ssh                = true,
+  Boolean $out_ntp               = true,
+  Boolean $out_dns               = true,
+  Boolean $out_http              = true,
+  Boolean $out_https             = true,
+  Boolean $out_all               = false,
+  Hash $rules                    = {},
+  String $log_prefix             = '[nftables] %<chain>s Rejected: ',
```

Comment:
I suppose to move `Rejected` to the `inet_filter.pp`. If the log prefix is used at another place where the packages are just logged and not rejected, the prefix is wrong. And the variable name is really generic.

Reply:
Thanks for your comment. Not sure I'm following, though. Are you suggesting to replace the default value of `$log_prefix` here by `[nftables] %<chain>s` and perhaps append the `Rejected:` bit [here](https://github.com/duritong/puppet-nftables/blob/master/manifests/inet_filter.pp#L61) where we now for sure that the packet will be discarded (either via a reject or a drop)? Makes sense.

Reply:
If so, would you mind if I did these changes in a separate MR?

Reply:
Yes, that's what I meant. Thank you for asking and clarification. You can implement that in another merge request, that's fine for me.